app/reducers/emojis.ts

```diff
@@ -1,25 +1,23 @@
-import { createSelector } from 'reselect';
+import { createSlice } from '@reduxjs/toolkit';
 import { Emoji } from 'app/actions/ActionTypes';
-import createEntityReducer from 'app/utils/createEntityReducer';
+import { EntityType } from 'app/store/models/entities';
+import createLegoAdapter from 'app/utils/legoAdapter/createLegoAdapter';
+import type { RootState } from 'app/store/createRootReducer';
 
-export const selectEmojis = createSelector(
-  (state) => state.emojis.byId,
-  (state) => state.emojis.items,
-  (emojisById, emojiIds) => {
-    return emojiIds.map((id) => emojisById[id]);
-  }
-);
-export const selectEmojisById = createSelector(
-  selectEmojis,
-  (state, emojisId) => emojisId,
-  (emojis, emojisId) => {
-    if (!emojis || !emojisId) return {};
-    return emojis.find((emojis) => emojis.shortCode === emojisId);
-  }
-);
-export default createEntityReducer({
-  key: 'emojis',
-  types: {
-    fetch: [Emoji.FETCH, Emoji.FETCH_ALL],
-  },
+const legoAdapter = createLegoAdapter(EntityType.Emojis, {
+  selectId: (emoji) => emoji.shortCode,
+});
+
+const emojisSlice = createSlice({
+  name: 'emojis',
```

Comment:
```suggestion
  name: EntityType.Emojis,
```

These are the only inconsistencies I can see

Reply:
Ah, right. Seems like I forgot this on a lot of them👍